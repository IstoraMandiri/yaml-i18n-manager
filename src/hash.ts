import { createHash } from 'crypto';

export default function (str: string): string {
  return createHash('sha256').update(str, 'utf8').digest('base64').slice(0, 10);
}
