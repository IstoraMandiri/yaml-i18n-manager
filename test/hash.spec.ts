import hash from '../src/hash';

describe('hash', () => {
  it('hashes', async () => {
    expect(hash('hello')).toMatchSnapshot();
  });
});
