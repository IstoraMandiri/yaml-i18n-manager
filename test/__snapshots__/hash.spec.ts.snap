// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hash hashes 1`] = `"LPJNul+wow"`;
